scripts/deploy.sh

```diff
@@ -60,11 +60,17 @@ invalidate-cloudfront() {
     echo "Creating invalidation"
     local -a paths_to_invalidate
     for file_path in "${FILE_PATHS[@]}"; do
-      paths_to_invalidate+=("/$(basename "$file_path")")
+      paths_to_invalidate+=($(suffixed-file-name "$file_path"))
```

Comment:
There was a slash at the beginning of the path, did it matter?

Reply:
I'll double check